mobile/src/main/java/rs/readahead/washington/mobile/util/FailedUnlockManager.kt

```diff
@@ -0,0 +1,42 @@
+package rs.readahead.washington.mobile.util
+
+import rs.readahead.washington.mobile.R
+import rs.readahead.washington.mobile.data.sharedpref.Preferences
+
+class FailedUnlockManager {
+
+    companion object {
+        const val OPTION_OFF = 0L
+        const val OPTION_5_ATTEMPTS = 5L
```

Comment:
Instead of using constants, you could use an enum class to define your unlock options. This will improve readability. You can also associate each option with its corresponding resource ID.
maybe something like this : 
enum class FailedUnlockOption(val value: Long, val resourceId: Int) {
    OFF(OPTION_OFF, R.string.Settings_Off_Do_Not_Delete),